enterprise_access/apps/bffs/serializers.py

```diff
@@ -152,12 +152,27 @@ class EnterpriseCustomerSerializer(BaseBffSerializer):
     show_integration_warning = serializers.BooleanField()
 
 
+class EnterpriseCustomerUsersSerializer(BaseBffSerializer):
+    """
+    Serializer for all linked enterprise customer users
+    """
+    id = serializers.IntegerField()
+    user_id = serializers.IntegerField()
+    enterprise_customer = EnterpriseCustomerSerializer()
+    active = serializers.BooleanField()
+
+
 class BaseResponseSerializer(BaseBffSerializer):
     """
     Serializer for base response.
     """
 
     enterprise_customer = EnterpriseCustomerSerializer(required=False, allow_null=True)
+    all_linked_enterprise_customer_users = serializers.ListField(
+        child=EnterpriseCustomerUsersSerializer(),
+        allow_empty=True,
+    )
```

Comment:
nit: as a sanity check, the following doesn't work?

```python
all_linked_enterprise_customer_users = EnterpriseCustomerUsersSerializer(many=True, allow_empty=True, default=list)
```

(FWIW, using the `many=True` approach seems to pass all tests locally still).